src/crawler.ts

```diff
@@ -2363,15 +2379,54 @@ self.__bx_behaviors.selectMainBehavior();
     }
   }
 
+  getWarcPrefix(defaultValue = "") {
```

Comment:
Maybe we should set `rec` as the default value here rather than in the calling below?

Reply:
sure, yes, for the auxiliary writers, they could pass in empty string.

Reply:
though, this will result in:
```
this.getWarcPrefix() meaning "rec"
this.getWarcPrefix("") meaning ""
```
not sure if its clearer this way -- could also remove the default value altogether.

Reply:
Ah right, eh, I think it's fine to leave as-is given the auxiliary writers, you make a good point.